<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="pc" failures="1" tests="3" name="Default test" time="125.305" errors="0" timestamp="20 Nov 2018 02:54:41 GMT">
  <testcase classname="com.demoaut.newtours.MavenDemo.BookTicketTest" name="ticketBookingTest" time="60.741"/>
  <testcase classname="com.demoaut.newtours.MavenDemo.InvalidUserTest" name="userValidation" time="1.827">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.demoaut.newtours.MavenDemo.InvalidUserTest.userValidation(InvalidUserTest.java:26)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- userValidation -->
  <testcase classname="com.demoaut.newtours.MavenDemo.ValidateUSerTest" name="userValidation" time="45.041"/>
  <testcase classname="com.demoaut.newtours.MavenDemo.BaseTest" name="@AfterSuite tearDown" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite tearDown -->
  <testcase classname="com.demoaut.newtours.MavenDemo.BaseTest" name="@AfterSuite tearDown" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite tearDown -->
  <testcase classname="com.demoaut.newtours.MavenDemo.BaseTest" name="@AfterSuite tearDown" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite tearDown -->
</testsuite> <!-- Default test -->
